services/repository/src/main/java/org/sagebionetworks/repo/web/controller/TeamController.java

```diff
@@ -138,7 +140,27 @@ Team createTeam(
 			) {
 		return serviceProvider.getTeamService().getByMember(id, limit, offset);
 	}
-	
+
+	/**
+	 * Retrieve a paginated list of IDs of Teams to which the given user belongs.
```

Comment:
If sorting is desired then both 'sortBy' and 'ascending' must be specified.  If they are omitted then no sort is applied.